crates/miden-lib/asm/account_components/multisig_rpo_falcon_512.masm

```diff
@@ -16,26 +16,31 @@ const.UNAUTHORIZED_EVENT=event("miden::auth::unauthorized")
 
 # Storage Layout Constants
 #
-# ┌─────────────────────────────┬──────────┬──────────────┐
-# │    THRESHOLD & APPROVERS    │ PUB KEYS │ EXECUTED TXS │
-# │           (slot)            │   (map)  │    (map)     │
-# ├─────────────────────────────┼──────────┼──────────────┤
-# │              0              │    1     │      2       │
-# └─────────────────────────────┴──────────┴──────────────┘
-
-# The slot in this component's storage layout where both the signature threshold
-# and number of approvers are stored as [threshold, num_approvers, 0, 0].
+# ┌───────────────────────────────┬──────────┬──────────────┬──────────────┐
+# │ THRESHOLD & APPROVERS CONFIG  │ PUB KEYS │ EXECUTED TXS │  PROC ROOTS  │
+# │           (slot)              │   (map)  │    (map)     │    (map)     │
+# ├───────────────────────────────┼──────────┼──────────────┼──────────────┤
+# │              0                │    1     │      2       │      3       │
+# └───────────────────────────────┴──────────┴──────────────┴──────────────┘
+
+# The slot in this component's storage layout where the signature threshold,
+# number of approvers are stored as
+# [threshold, num_approvers, 0, 0].
```

Comment:
Should we refer to `threshold` as `default_threshold` (or something similar)? Also, would be good to explain this concept somewhere.